.vscode/settings.json

```
@@ -24,8 +19,35 @@
         "url": "http://localhost:4000",
         "webRoot": "${workspaceFolder}",
         "userDataDir": "${workspaceFolder}/.vscode/chrome-debug-profile"
+      },
+      {
+        "type": "node",
+        "request": "attach",
+        "name": "Attach to Node Process",
+        "port": 9229
+      },
+      {
+        "name": "Debug Jest Tests",
+        "type": "node",
+        "request": "launch",
+        "runtimeArgs": [
+          "--inspect-brk",
+          "${workspaceRoot}/node_modules/.bin/jest",
+          "--runInBand",
+          "--watch",
+          "${input:pickFile}"
+        ],
```

You can do this I believe so that it just runs the npm script and will stay in sync if that changes for some reason

```suggestion
        "runtimeExecutable": "npm",
        "runtimeArgs": ["run-script", "test:debug", "${input:pickFile}"],
```

---

Not sure why, but running npm here always auto breaks inside the jest module which is kind of annoying. Going to leave it as is for better UX. Should be rare the script would get changed anyway